idb/postgres/internal/encoding/encode.go

```diff
@@ -60,6 +96,11 @@ func convertSignedTxnWithAD(stxn types.SignedTxnWithAD) types.SignedTxnWithAD {
 
 // EncodeSignedTxnWithAD returns a json string where all byte arrays are base64 encoded.
 func EncodeSignedTxnWithAD(stxn types.SignedTxnWithAD) []byte {
+	ap := stxn.Txn.AssetParams
+	ap.AssetName = strings.ReplaceAll(ap.AssetName, "\000", "")
```

Comment:
leftover from a previous incarnation?

Reply:
Kind of. This is used for encoding the transaction JSON, so these fields are the "convenience" fields, we don't actually use them directly. Since there is no b64 equivalent in this data structure I did the minimal processing. This field would only be used by people poking around in the DB manually.

Reply:
Is it really possible to have this ? i.e. the AssetName should not contain these characters, so I'm kind of puzzled why we want to have this code.

Reply:
The strings need to be sanitized in the Asset table and the Transaction table.

Based on a discussion with Tokil I'm modifying the encoding to match some conventions used in the accounting rewrite and this special case is going away.